astropy/io/fits/scripts/fitscheck.py

```diff
@@ -40,72 +40,77 @@
 """
 
 
+import argparse
 import logging
-import optparse
 import sys
-import textwrap
 
 from astropy.tests.helper import catch_warnings
 from astropy.io import fits
 
 
 log = logging.getLogger('fitscheck')
 
+_DESCRIPTION = """
+.e.g. fitscheck example.fits
```

Comment:
What was the leading period for? Seems like bad grammar. Did I miss something?
```suggestion
e.g. fitscheck example.fits
```

Reply:
Hm... 

maybe to make it more symmetric? 🤔

Next time I'll be more careful when copy&pasting.

Reply:
I'll remove that once I know if I should add a changelog, okay?

Reply:
I am leaning on no need for change log but I'll let Simon chime in. Sure, we can wait. And I don't think it is your fault, as the leading period was already there before but I just never noticed it until the diff highlighted it. :sweat_smile:

Reply:
I would completely remove this sentence, it is useless as it is redundant with the help formatted by optparse/argparse. This is for example the current one with optparse:
```
Usage: fitscheck [options] <.fits files...>

        .e.g. fitscheck example.fits

        Verifies and optionally re-writes the CHECKSUM and DATASUM keywords
        for a .fits file.
        Optionally detects and fixes FITS standard compliance problems.
```

Reply:
The output (especially the usage) with this PR is somewhat different:

```
$ fitscheck -h
usage: fitscheck [-h] [-k {standard,remove,none}] [-w] [-f] [-c] [-i] [-v]
                 file [file ...]

.e.g. fitscheck example.fits

Verifies and optionally re-writes the CHECKSUM and DATASUM keywords
for a .fits file.
Optionally detects and fixes FITS standard compliance problems.

positional arguments:
  file                  .fits files to process.

optional arguments:
  -h, --help            show this help message and exit
  -k {standard,remove,none}, --checksum {standard,remove,none}
                        Choose FITS checksum mode or none. Defaults standard.
  -w, --write           Write out file checksums and/or FITS compliance fixes.
  -f, --force           Do file update even if original checksum was bad.
  -c, --compliance      Do FITS compliance checking; fix if possible.
  -i, --ignore-missing  Ignore missing checksums.
  -v, --verbose         Generate extra output.
```